### PR TITLE
[openshift-rolebindings] support cluster admin

### PR DESCRIPTION
**Pull request by maorfr**
related to https://issues.redhat.com/browse/APPSRE-6737

to be able to manage RoleBinding objects in protected namespaces as cluster admin

### CODE REVIEW

File: reconcile/openshift_rolebindings.py [hunk 1]
```diff
@@ -112,7 +113,9 @@ def fetch_desired_state(ri, oc_map):
         for permission in permissions:
             cluster_info = permission["cluster"]
             cluster = cluster_info["name"]
-            namespace = permission["namespace"]
+            namespace_info = permission["namespace"]
+            namespace = namespace_info["name"]
```
Comment (janboll) [inline reconcile/openshift_rolebindings.py:117]: Wow, the usage of the word namespace in this method is quite confusing :sweat: 
What about renaming this to `perm_namespace_name`

File: reconcile/openshift_rolebindings.py [hunk 2]
```diff
@@ -146,17 +150,18 @@ def fetch_desired_state(ri, oc_map):
             for sa in service_accounts:
                 if ri is None:
                     continue
-                namespace, sa_name = sa.split("/")
+                sa_namespace, sa_name = sa.split("/")
```
Comment (janboll) [inline reconcile/openshift_rolebindings.py:153]: What about renaming this to `sa_namespace_name`

File: reconcile/openshift_rolebindings.py [hunk 3]
```diff
@@ -112,7 +113,9 @@ def fetch_desired_state(ri, oc_map):
         for permission in permissions:
             cluster_info = permission["cluster"]
             cluster = cluster_info["name"]
-            namespace = permission["namespace"]
+            namespace_info = permission["namespace"]
```
Comment (janboll) [inline reconcile/openshift_rolebindings.py:116]: what about renaming this to `perm_namespace`?

Comment (maorfr) [reply reconcile/openshift_rolebindings.py:116]: i did 2/3. `namespace_info` is quite common around the codebase.

**Review by janboll — APPROVED** [review]

**Pull request merged**